resolwe/elastic/builder.py

```diff
@@ -98,12 +183,18 @@ def __init__(self, field):
         """Construct m2m dependency."""
         super(ManyToManyDependency, self).__init__(field.rel.model)
         self.field = field
+        self.delete_cache = BuildArgumentsCache()
+        self.remove_cache = BuildArgumentsCache()
```

Comment:
The naming is non-obvious. Delete vs. remove? Can this be documented somewhere or renamed?